PackageConfig.json

```diff
@@ -16,6 +16,160 @@
         "DP-Name1",
         "DP-Name2"
     ],
+    "collectionTargets": [
```

Comment:
Do we want this to be custom per app or should we have some kind of standard structure that can be defined in the global config?  Maybe go with this for now since it works but I want this moved into global personally.

Reply:
I think it could work in the global with some tweaks. The reason I put it in the packageconfig was that maybe people would want custom deployment settings per collection per application.